Nodejs/Product/TestAdapterImpl/TestContainerDiscoverer.cs

```diff
@@ -289,6 +290,11 @@ public IEnumerable<ITestContainer> GetTestContainers(IVsProject project)
                     return latest;
                 });
 
+            if (!string.IsNullOrEmpty(testFramework))
+            { 
+                Microsoft.NodejsTools.Telemetry.TelemetryHelper.LogTestDiscoveryStarted(testFramework);
```

Comment:
Is this still on starting a discovery? if not, I would change the name of the log

Reply:
It's debatable :). This gets the test containers and that happens on discovery (execution already has them because of discovery). The most accurate would be "Container Discovery Started" but that's mostly an implementation detail. I prefer to have it as Discovery Started in case I can shift it to the right location in the container.